src/lib/generator.ts

```diff
@@ -212,17 +213,48 @@ function formatConversions(build: BuildSettings): string[] {
   return formatTable(headers, body);
 }
 
-function formatTiresAndRims(build: BuildSettings): string[] {
-  return formatTable(['Tires And Rims', ''], [
-    ['Compound', build.tiresAndRims.compound],
-    ['Tire Width', `Front ${build.tiresAndRims.width.front} mm, Rear ${build.tiresAndRims.width.rear} mm`],
-    ['Rim Style', `${build.tiresAndRims.rimStyle.type} ${build.tiresAndRims.rimStyle.name}`],
-    ['Rim Size', `Front ${build.tiresAndRims.rimSize.front} in, Rear ${build.tiresAndRims.rimSize.rear} in`],
-    ['Track Width', `Front ${build.tiresAndRims.trackWidth.front}, Rear ${build.tiresAndRims.trackWidth.rear}`],
-  ]);
-}
+export function formatBuild(build: BuildSettings): string[] {
+  const text = [
+    ...formatConversions(build),
+    ...formatTable(['Engine', ''], [
+      ['Intake', build.engine.intake],
+      ['Fuel System', build.engine.fuelSystem],
+      ['Ignition', build.engine.ignition],
+      ['Exhaust', build.engine.exhaust],
+      ['Camshaft', build.engine.camshaft],
+      ['Valves', build.engine.valves],
+      ['Displacement', build.engine.displacement],
+      ['Pistons', build.engine.pistons],
+      ['Turbo', build.engine.turbo],
+      ['Twin Turbo', build.engine.twinTurbo],
+      ['Supercharger', build.engine.supercharger],
+      ['Centrifigul Supercharger', build.engine.centrifigulSupercharger],
+      ['Intercooler', build.engine.intercooler],
+      ['Oil Cooling', build.engine.oilCooling],
+      ['Flywheel', build.engine.flywheel],
+    ].filter((v) => v[1] !== Upgrade.na)),
+    ...formatTable(['Platform And Handling', ''], [
+      ['Brakes', build.platformAndHandling.brakes],
+      ['Springs', build.platformAndHandling.springs],
+      ['Front Arb', build.platformAndHandling.frontArb],
+      ['Rear Arb', build.platformAndHandling.rearArb],
+      ['Chassis Reinforcement', build.platformAndHandling.chassisReinforcement],
+      ['Weight Reduction', build.platformAndHandling.weightReduction],
+    ]),
+    ...formatTable(['Drivetrain', ''], [
+      ['Clutch', build.drivetrain.clutch],
+      ['Transmission', build.drivetrain.transmission],
+      ['Driveline', build.drivetrain.driveline],
+      ['Differential', build.drivetrain.differential],
+    ]),
+    ...formatTable(['Tires And Rims', ''], [
+      ['Compound', build.tiresAndRims.compound],
+      ['Width', `Front ${build.tiresAndRims.width.front} mm, Rear ${build.tiresAndRims.width.rear} mm`],
+      ['Rim Style', `${build.tiresAndRims.rimStyle.type} ${build.tiresAndRims.rimStyle.name}`],
+      ['Rim Size', `Front ${build.tiresAndRims.rimSize.front} in, Rear ${build.tiresAndRims.rimSize.rear} in`],
+    ]),
```

Comment:
Also moved from a separate function.  In this case Track Width was removed as well.  Reasoning?